src/components/table/table-row/table-row.js

```diff
@@ -170,37 +196,37 @@ class TableRow extends React.Component {
   }
 
   /**
-   * @method componentWillUnmount
-   * @return {Void}
-   */
-  componentWillUnmount() {
-    if (this.context.detachFromTable) {
-      this.context.detachFromTable(this.rowID);
-    }
-  }
-
-  /**
    * @method componentWillReceiveProps
    * @return {Void}
    */
   componentWillReceiveProps(nextProps) {
-    if (this.props.uniqueID != nextProps.uniqueID) {
+    if (this.props.uniqueID !== nextProps.uniqueID) {
```

Comment:
Yeah it's a good point. Currently the linter is throwing an error on this. Not sure if there's an alternative implementation or if the rule should be removed?